packages/round-manager/src/features/round/ViewRoundSettings.tsx

```diff
@@ -122,6 +122,43 @@ export default function ViewRoundSettings(props: { id?: string }) {
   const [ipfsStep, setIpfsStep] = useState(false);
   const [hasChanged, setHasChanged] = useState(false);
 
+  const [noRoundEndDate, setNoRoundEndDate] = useState(false);
+  // Is the round a rolling application round as per the contract
+  const [rollingApplications, setRollingApplications] = useState(false);
+  // Is the round a rolling application round as per the form
+  const [editedRollingApplications, setEditedRollingApplications] =
+    useState(false);
+
+  useEffect(() => {
+    setNoRoundEndDate(moment(round?.roundEndTime).isSame(maxDateForUint256));
+  }, [round?.roundEndTime]);
+
+  useEffect(() => {
+    if (
+      round?.roundEndTime.toISOString() !== "" &&
+      round?.applicationsEndTime.toISOString() ===
+        round?.roundEndTime.toISOString()
+    ) {
+      setRollingApplications(true);
+    }
+  }, [round?.applicationsEndTime, round?.roundEndTime]);
+
+  useEffect(() => {
+    if (
+      editedRound?.roundEndTime.toISOString() !== "" &&
+      editedRound?.applicationsEndTime.toISOString() ===
+        editedRound?.roundEndTime.toISOString()
+    ) {
+      setEditedRollingApplications(true);
+    }
+  }, [editedRound?.applicationsEndTime, editedRound?.roundEndTime]);
+
+  useEffect(() => {
+    if (isDirectRound(round!)) {
+      setEditedRollingApplications(true);
+    }
+  }, [round]);
+
```

Comment:
Could be a good idea to move this into a hook. If a lot of these states are form states we might want to add react-hook-form to handle that for us.

Reply:
Will be fixed as part of round edit flow code refactoring ticket.